aspnetcore/fundamentals/websockets.md

```diff
@@ -41,16 +42,9 @@ This article explains how to get started with WebSockets in ASP.NET Core. [WebSo
 
 Use WebSockets to work directly with a socket connection. For example, use WebSockets for the best possible performance with a real-time game.
 
-[ASP.NET SignalR](/aspnet/signalr/overview/getting-started/introduction-to-signalr) provides a richer app model for real-time functionality, but it only runs on ASP.NET 4.x, not ASP.NET Core. An ASP.NET Core version of SignalR is scheduled for release with ASP.NET Core 2.1. See [ASP.NET Core 2.1 high-level planning](https://github.com/aspnet/Announcements/issues/288).
+[ASP.NET Core SignalR](xref:signalr/introduction) is a library that simplifies adding real-time web functionality to apps, and uses the WebSockets protocol whenever possible. SignalR is useful for most development scenarios.
```

Comment:
- Break the 1st sentence into 2. The 2nd sentence could read "It uses the WebSockets...".
- This last sentence is a bit vague. I recommend either removing it or adding a bit more detail.

Reply:
I'd prefer the term "WebSockets" alone over "WebSockets protocol". WebSockets is a bit more than a protocol, and is sufficiently descriptive on it's own IMO